packages/amplify-codegen/src/commands/add.js

```diff
@@ -80,13 +80,11 @@ async function add(context, apiId = null) {
     excludes: answer.excludePattern,
     schema,
     amplifyExtension: {
-      graphQLApiId: apiDetails.id,
       codeGenTarget: answer.target || '',
       generatedFileName: answer.generatedFileName || '',
       docsFilePath: answer.docsFilePath,
       region,
     },
-    endpoint: apiDetails.endpoint,
```

Comment:
Did you use the endpoint anywhere in the statement or doc generation? or was it just the appID?

Reply:
We don't use it anywhere. Even with the endpoint present user will have to do some custom headers to support the Auth for AppSync.